### DISCUSSION TITLE
第六节课笔记（17班）

### DISCUSSION THREAD

**Discussion by Jianfeng777** [InternLM/Tutorial#449] [笔记]
请上传第六节课的笔记在这里！

**Comment by l241025097** [comment]
第六节课笔记
![06大模型评测OpenCompass读书笔记](https://github.com/InternLM/tutorial/assets/31737569/52b5fc3e-d1bf-4f08-9add-9d2b761805fa)

**Comment by fine-lab** [comment]
https://blog.csdn.net/SmallZCan_t/article/details/135743882?spm=1001.2014.3001.5501

**Comment by zcg-joker** [comment]
[第六节课笔记](https://blog.csdn.net/m0_55764641/article/details/135754414)

**Comment by AxrLH** [comment]
class 6：
https://blog.csdn.net/Axr_Samuel/article/details/135731462?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135731462%22%2C%22source%22%3A%22Axr_Samuel%22%7D

**Comment by futtttttttttttt** [comment]
https://zhuanlan.zhihu.com/p/679749059
